test/extensions/INTEL/SPV_INTEL_joint_matrix/joint_matrix_bfloat16_offset.ll

```diff
@@ -0,0 +1,616 @@
+  %14 = bitcast i8* %arrayinit.element1 to i64*
+  store i64 %13, i64* %14, align 8
+  %15 = call spir_func i64 @_Z29__spirv_BuiltInGlobalLinearIdv() #5
+  call spir_func void @__itt_offload_wi_finish_stub([3 x i64] addrspace(4)* %GroupID.ascast, i64 %15) #3
+  %16 = bitcast [3 x i64]* %GroupID to i8*
+  call void @llvm.lifetime.end.p0i8(i64 24, i8* %16)
+  br label %return
+
+return:                                           ; preds = %if.end, %entry
+  ret void
+}
+
+; Function Attrs: noinline nounwind optnone
+define internal spir_func void @__itt_offload_wi_finish_stub([3 x i64] addrspace(4)* %group_id, i64 %wi_id) #3 {
+entry:
+  %group_id.addr = alloca [3 x i64] addrspace(4)*, align 8, !spirv.Decorations !339
+  %wi_id.addr = alloca i64, align 8, !spirv.Decorations !339
+  %group_id.addr.ascast = addrspacecast [3 x i64] addrspace(4)** %group_id.addr to [3 x i64] addrspace(4)* addrspace(4)*
+  %wi_id.addr.ascast = addrspacecast i64* %wi_id.addr to i64 addrspace(4)*
+  store [3 x i64] addrspace(4)* %group_id, [3 x i64] addrspace(4)* addrspace(4)* %group_id.addr.ascast, align 8
+  store i64 %wi_id, i64 addrspace(4)* %wi_id.addr.ascast, align 8
+  ret void
+}
+
+; Function Attrs: nounwind readnone willreturn
+declare spir_func i64 @_Z33__spirv_BuiltInGlobalInvocationIdi(i32) #5
+
+; Function Attrs: nounwind readnone willreturn
+declare spir_func i64 @_Z32__spirv_BuiltInLocalInvocationIdi(i32) #5
+
+; Function Attrs: nounwind readnone willreturn
+declare spir_func i64 @_Z26__spirv_BuiltInWorkgroupIdi(i32) #5
+
+; Function Attrs: nounwind readnone willreturn
+declare spir_func i64 @_Z29__spirv_BuiltInGlobalLinearIdv() #5
+
+; Function Attrs: nounwind readnone willreturn
+declare spir_func i64 @_Z28__spirv_BuiltInWorkgroupSizei(i32) #5
+
+attributes #0 = { nounwind }
+attributes #1 = { alwaysinline nounwind }
+attributes #2 = { argmemonly nofree nosync nounwind willreturn }
+attributes #3 = { noinline nounwind optnone }
+attributes #4 = { inaccessiblememonly nofree nosync nounwind willreturn }
+attributes #5 = { nounwind readnone willreturn }
+
+!spirv.MemoryModel = !{!0}
+!spirv.Source = !{!1}
+!opencl.spir.version = !{!2}
+!opencl.ocl.version = !{!3}
+!opencl.used.extensions = !{!4}
+!opencl.used.optional.core.features = !{!4}
+!spirv.Generator = !{!5}
+!opencl.compiler.options = !{!6}
+!igc.functions = !{}
+!IGCMetadata = !{!7}
+
+!0 = !{i32 2, i32 2}
+!1 = !{i32 4, i32 100000}
+!2 = !{i32 1, i32 2}
+!3 = !{i32 1, i32 0}
+!4 = !{}
+!5 = !{i16 6, i16 14}
+!6 = !{!"-ze-opt-level=O2"}
+!7 = !{!"ModuleMD", !8, !9, !111, !112, !143, !144, !148, !151, !152, !153, !189, !215, !228, !229, !230, !246, !247, !248, !249, !250, !251, !252, !253, !254, !255, !259, !260, !267, !268, !269, !270, !271, !272, !273, !274, !275, !276, !277, !278, !280, !284, !285, !286, !287, !288, !289, !290, !291, !292, !293, !294, !295, !296, !297, !298, !299, !300, !301, !302, !303, !304, !305, !307, !310, !311, !312, !314, !315, !316, !321}
```

Comment:
Please shorten the tests - we won't need all of these metadata

Reply:
Thanks for the suggestions! I have updated the test.